gulpfile.js

```diff
@@ -182,6 +183,28 @@ gulp.task('compile:sitemap', 'generate sitemap.xml', function() {
       .pipe(gulp.dest(paths.dist.dir));
 });
 
+gulp.task('compile:sw-precache',
+          ['copy:images', 'copy:videos', 'compile:example'], function() {
+            const staticDir = 'static';
+
+            swPrecache.write(path.join(staticDir, 'sw.js'), {
+              staticFileGlobs: [
+                path.join(paths.dist.dir, 'LICENSE.txt'),
+                path.join(paths.dist.img, 'gist.png'),
+                path.join(paths.dist.img, 'abe_preview.png'),
+                path.join(paths.dist.favicons, '*.png'),
+                path.join(paths.dist.dir,
+                   'components/amp-install-serviceworker/*.html')
```

Comment:
Do you mean fonts under https://fonts.googleapis.com/ ? Service Worker can't access resources out side of its scope.